modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/resources/content/Language.properties

```diff
@@ -12,17 +12,18 @@ go-to-workspace=Go to Workspace
 invite-teammates-to-collaborate=Invite teammates to collaborate
 log-in-to-your-workspace-to-access-this-property=Log in to your workspace to access this property.
 member-fragment=member
-new-request-to-join-workspace=New Request to Join Workspace
```

Comment:
Hey @ivicac, `Language.properties` changes should be in a different commit than the changes in generated files